Xamarin.PropertyEditing.Mac/Xamarin.PropertyEditing.Mac.csproj

```diff
@@ -18,6 +18,9 @@
     <ProjectReference Include="..\Xamarin.PropertyEditing\Xamarin.PropertyEditing.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Remove="Controls\Custom\UnderlinedTextField.cs" />

```

Comment:
This looks like it shouldn't have been serialized in the csproj.

Reply:
Correct.